src/commands/scorecard.js

```diff
@@ -0,0 +1,104 @@
+// Import Node.js Dependencies
+import fs from "fs";
+
+// Import Third-party Dependencies
+import cliui from "@topcli/cliui";
+import kleur from "kleur";
+import * as scorecard from "@nodesecure/ossf-scorecard-sdk";
+import ini from "ini";
+
+// VARS
+const { yellow, grey, cyan, white } = kleur;
+
+function separatorLine() {
+  return grey("-".repeat(80));
+}
+
+export function getCurrentRepository() {
```

Comment:
Using an error pattern to deal with error if probably better here

```js
const result = getCurrentRepository();
if (!result.ok) {
  return console.log(kleur.white().bold(result.reason));
}

const repository = result.value;
```

You handle the log and exit of command at higher level (eventually allow you to write UT for that function too).

Reply:
Indeed its done, we'll see for ut